aptos-move/vm-genesis/src/lib.rs

```diff
@@ -417,12 +417,12 @@ fn initialize(
     // Calculate the per-epoch rewards rate, represented as 2 separate ints (numerator and
     // denominator).
     let rewards_rate_denominator = 1_000_000_000;
-    // Fold division of the percentage by 100 into multiplication with the denominator
-    // to minimize rounding errors due to integer division.
-    let rewards_rate_numerator = genesis_config.rewards_apy_percentage
-        * (rewards_rate_denominator / 100)
-        * genesis_config.epoch_duration_secs
-        / NUM_SECONDS_PER_YEAR;
+    let _num_epochs_in_a_year = NUM_SECONDS_PER_YEAR / genesis_config.epoch_duration_secs;
```

Comment:
This will still trigger division by zero if we set `epoch_duration_secs` to a value longer than a year, so perhaps comment this out completely.
The code I replaced it with computes the numerator correctly for a wider range of values.